### PR TITLE
Feature/its a stg

### PR DESCRIPTION
**Pull request by hirudan**
- Add player shots
- Add basic enemy shots (which we'll probably replace)
- Bullet frequency still needs tweaking, but that's just changing constants  until we're happy

### CODE REVIEW

File: src/entity/player.js [hunk 1]
```diff
@@ -47,19 +57,34 @@ class Player extends Entity  {
         return sprite;
     }
 
+    damage() {
+        console.log("itai itai itai");
```
Comment (ProgrammingIncluded) [inline src/entity/player.js]: Please use `G_LOGGER.debug()` if possible.

File: src/entity/player.js [hunk 2]
```diff
@@ -47,19 +57,34 @@ class Player extends Entity  {
...
         // NESW movement
-        let posBuf = this.movement(this.elapsed, this.container.position, undefined, undefined, undefined);
+        let posBuf = this.movement(undefined, this.container.position, undefined, undefined, undefined);
         this.container.position = posBuf;
         // Rotate towards mouse location
         // Convert mouse target point to game field coordinate system
         this.lookTowards(this.parent.toLocal(G_PIXI_APP.renderer.plugins.interaction.mouse.global));
+        // Hit testing
+        // Shooting
+        if(this.keysPressed["space"]) {
```
Comment (ProgrammingIncluded) [inline src/entity/player.js:85]: Did we want to change to this RMB?

**Review by ProgrammingIncluded — APPROVED** [review]
Great job overall!

File: src/entity/player.js (hunk 2 above)
Comment (hirudan) [reply src/entity/player.js:85]: Hah I thought per Discord discussion we wanted to leave this and related changes for later.

**Pull request merged**